### PR TITLE
[DED-113] Fix 404 and broken links in the docs

### PR DESCRIPTION
**Pull request by oliwiapolec**
# Deploy preview
For the time being, deploy previews are generated using the Nelify CLI. More info on Slack (#plaform-docs).

# Links
- [Jira](https://livechatinc.atlassian.net/browse/DED-113)

# Description
Added redirects for 404 links from [this document](https://docs.google.com/spreadsheets/d/1l863BPCevayKOfGl7QGgTdLNV_XospMpFy2I6BfMPFk/edit#gid=0) and fixed broken links in the docs.

### CODE REVIEW

**Comment by ale-kacperczyk** [comment]
https://6241b5da4b4933123b4e912d--livechat-public-docs.netlify.app

File: _redirects [hunk 1]
```diff
@@ -38,7 +38,21 @@
...
-/messaging/apple-business-chat/*    /docs/messaging/apple-messages-for-business/:splat  301 
+/messaging/apple-business-chat/*    /docs/messaging/apple-messages-for-business/:splat  301
+/changelog/  /docs/messaging/ 301
+/chat-widget-moments/   /docs/extending-chat-widget/chat-widget-moments/ 301
+/chat-widget-themes/   /docs/extending-chat-widget-customer-sdk 301
+/management/getting-started/guides/webhook-apps/*    /docs/getting-started/livechat-apps/webhook-apps/:splat 301
+/management/webhooks/changelog/*    /docs/management/changelog/:splat 301
+/management/webhooks/v3.3/changelog/*    /docs/management/changelog/:splat 301
+/messaging/rtm-reference/   /docs/messaging/ 301
+/messaging/agent-chat-api/v3.3/rtm-reference/messaging/agent-chat-api/rtm-pushes/    /docs/messaging/agent-chat-api/rtm-pushes 301
```
Comment (ale-kacperczyk) [inline _redirects:49]: @jakubsikora, is it OK to have this many "edge case" redirects?

Comment (jakubsikora) [reply _redirects:49]: Yep no problem

**Review by jakubsikora — APPROVED** [review]
Looks good.

**Pull request merged**